es/installation/docker.md

```diff
@@ -46,7 +46,8 @@ sudo chmod +x /usr/local/bin/docker-compose
 
 Pendiente de ser completado... ¡Se agradecen las Contribuciones!
 
-## Instalación
+<!-- ## Instalación -->
```

Comment:
It looks like this comment was kept accidentally :thinking:.

Reply:
Solved! and rebased with master to remove conflicts!